components/cli/pkg/ballerina/docker_bal_executor.go

```diff
@@ -272,3 +275,107 @@ func (balExecutor *DockerBalExecutor) Version() (string, error) {
 func (balExecutor *DockerBalExecutor) ExecutablePath() (string, error) {
 	return "", nil
 }
+
+func (balExecutor *DockerBalExecutor) Init(projectDir string) error {
+	cliUser, err := user.Current()
+	if err != nil {
+		return fmt.Errorf("error while retrieving the current user, %v", err)
+	}
+	currentDir, err := os.Getwd()
+	if err != nil {
+		return fmt.Errorf("error in determining working directory, %v", err)
+	}
+	err = util.CleanAndCreateDir(filepath.Join(currentDir, constants.TARGET_DIR_NAME))
+	if err != nil {
+		return err
+	}
+	dockerCmdArgs := []string{"-u", cliUser.Uid,
+		"-l", "ballerina-runtime=" + version.BuildVersion(),
+		"--mount", "type=bind,source=" + filepath.Join(currentDir, constants.TARGET_DIR_NAME) + ",target=/home/cellery/tmp",
```

Comment:
Change the constant TARGET_DIR_NAME format from snake case to camel case.

Reply:
Fixed